demos/nbody/sim.hpp

```diff
@@ -131,12 +133,12 @@ class GravSim {
   integrator_t m_integrator;
 
   // Base constructor, does not initialize simulation values
-  GravSim(size_t n_bodies)
-      : m_q(sycl::default_selector{}, except_handler),
+  GravSim(size_t n_bodies) : 
```

Comment:
I don't really expect any diffs to be showing up here unless the files aren't formatted correctly at the moment

Reply:
Fixed